rplugin/python3/deoplete/sources/deoplete_jedi/worker.py

```diff
@@ -62,13 +62,10 @@ def completion_work(self, cache_key, extra_modules, source, line, col,
     def run(self):
         while True:
             try:
-                work = self.in_queue.get(block=False, timeout=0.5)
+                work = self.in_queue.get()
                 self.log.debug('Got work')
-                self.out_queue.put(self.completion_work(*work), block=False)
+                self.out_queue.put(self.completion_work(*work), timeout=0.5)
```

Comment:
I think this change does nothing, but I could be wrong.  I used `block=False` to keep the source verbose and intent clear, but I'm pretty sure it would never block since the queue size isn't fixed.